frontend/src/components/maps/leaflet/LayersControl/LayersControl.tsx

```diff
@@ -78,27 +86,33 @@ const ControlButton = styled(Button)`
 const LayersControl: React.FC = () => {
   const [open, setOpen] = React.useState<boolean>(false);
 
+  useEffect(() => {
+    const elem = L.DomUtil.get('layersContainer');
+    if (elem) {
+      L.DomEvent.on(elem!, 'mousewheel', L.DomEvent.stopPropagation);
```

Comment:
oh, so without this the map scrolls in the background if you scroll on the layers container?

Reply:
Yes sir